Source/Core/DolphinQt/GrandSettingsWindow.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <QDialogButtonBox>
+#include <QGridLayout>
+#include <QLineEdit>
+#include <QSortFilterProxyModel>
+#include <QStackedWidget>
+#include <QStringLiteral>
```

Comment:
This doesn't exist, it should just be `#include <QString>`.

Reply:
Thanks.  Yeah, it does seem to exist for some platforms, ex:  https://github.com/dolphin-emu/ext-win-qt/blob/a14e340be1ad66e0cb1d8180485896a9134800e0/Qt5.14.1/msvc2019_64/include/QtCore/qstringliteral.h

But a quick search shows that Qt mentions `QString` I'll move to that